packages/ethereum-contracts/test/foundry/apps/SuperAppBaseFlow.t.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity 0.8.16;
+
+import "forge-std/console.sol";
+import "../FoundrySuperfluidTester.sol";
+import { SuperAppBaseFlow } from "@superfluid-finance/ethereum-contracts/contracts/apps/SuperAppBaseFlow.sol";
+import { SuperAppBaseFlowTester } from "@superfluid-finance/ethereum-contracts/contracts/mocks/SuperAppBaseFlowTester.sol";
+import { ISuperToken } from "@superfluid-finance/ethereum-contracts/contracts/interfaces/superfluid/ISuperfluid.sol";
+import { IConstantFlowAgreementV1 } from "@superfluid-finance/ethereum-contracts/contracts/interfaces/agreements/IConstantFlowAgreementV1.sol";
+import { SuperTokenV1Library } from "@superfluid-finance/ethereum-contracts/contracts/apps/SuperTokenV1Library.sol";
+
+contract SuperAppBaseFlowTest is FoundrySuperfluidTester {
```

Comment:
first, start a nix shell in order to have the required tooling:
```
nix develop
```
(see CONTRIBUTING.md for how to set up Nix)

Then you can start a TDD session with 
```
nodemon -w test -w contracts/ -e sol --exec forge test --match-contract SuperAppBaseFlowTest -vvv
```